torch/fx/graph_module.py

```diff
@@ -95,17 +95,23 @@ def __init__(self, body):
 
     # Try to retrieve the forward source in a backward-compatible way
     CodeOnlyModule.forward = forward
+    com = CodeOnlyModule(body)
 
-    from .symbolic_trace import Tracer
+    if 'tracer' not in body:
```

Comment:
maybe add a note that this is strictly for BC at this point.

Reply:
Graphs can be constructed by hand too, right? Such graphs won't have a tracer.

But yes this is mostly for BC.